libpromises/policy.c

```diff
@@ -2312,7 +2313,8 @@ static Bundle *PolicyAppendBundleJson(Policy *policy, JsonElement *json_bundle)
         }
     }
 
-    Bundle *bundle = PolicyAppendBundle(policy, ns, name, type, args, source_path);
+    // TODO: add eval order in json
```

Comment:
Maybe you can link this to a ticket?

Reply:
Looks like you're adding it in the next commit. Then you should probably remove the comment?